apps/native/app/(tabs)/settings/index.tsx

```diff
@@ -1,11 +1,135 @@
 import React from "react";
-import { StyleSheet, Text, View } from "react-native";
+import {
+  StyleSheet,
+  Text,
+  View,
+  SafeAreaView,
+  ScrollView,
+  TouchableOpacity,
+  Pressable,
+} from "react-native";
+import { MaterialIcons } from "@expo/vector-icons";
+import { Colors } from "@/constants/Colors";
+
+const settingsItems = [
+  {
+    id: "account",
+    title: "Account",
+    icon: "person",
+    route: "/settings/account",
+  },
+  {
+    id: "notifications",
+    title: "Notifications",
+    icon: "notifications",
+    route: "/settings/notifications",
+  },
+  {
+    id: "privacy",
+    title: "Data & Privacy",
+    icon: "security",
+    route: "/settings/privacy",
+  },
+  {
+    id: "legal",
+    title: "Legal",
+    icon: "description",
+    route: "/settings/legal",
+  },
+  {
+    id: "support",
+    title: "Support",
+    icon: "help-outline",
+    route: "/settings/support",
+  },
+  {
+    id: "about",
+    title: "About",
+    icon: "info-outline",
+    route: "/settings/about",
+  },
+  {
+    id: "feedback",
+    title: "Send feedback",
+    subtitle: "App version 1.0",
+    icon: "feedback",
+    route: "/settings/feedback",
+  },
+];
+
+const SettingsItem = ({
+  item,
+  onPress,
+}: {
+  item: typeof settingsItems[0];
+  onPress: () => void;
+}) => {
+  return (
+    <Pressable style={styles.settingsItem} onPress={onPress}>
```

Comment:
Add android_ripple or style={({ pressed }) => ...} on Pressable for tactile feedback